pyiron_gui/project/project_browser.py

```diff
@@ -140,35 +272,68 @@ def __getattr__(self, item):
     def clear_output(self, *args, **kwargs):
         clear_button = kwargs.pop('clear_button', False)
         if clear_button:
-            self.buttons = widgets.HBox()
+            self.header = widgets.HBox()
         self.output.clear_output(*args, **kwargs)
         self.refresh()
 
-    def _update_buttons(self, obj):
+    def _click_button(self, b):
+        self.output.clear_output()
+        if b.description == "Show data":
```

Comment:
I get super uncomfortable as soon as logic is controlled by strings.

I'm not yet super familiar with how this code is working, but is it possible to readjust things so the button itself gets more power and this is a simple interface method?

```python
def _click_button(self, b: Button):
    self.output.clear_output()
    b.on_click()  # I'm stealing this method name directly from Unity3D default button object methods ;)
    self.display(obj=b.obj)
```